UserService/UserService.Repo/Migrations/20210317091314_AddBinAssistanceToOtherActivities.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace UserService.Repo.Migrations
+{
+    public partial class AddBinAssistanceToOtherActivities : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.InsertData(
```

Comment:
I would have thought there might already be a migration for these, but maybe we haven't updated the User Service in a while...